### PR TITLE
1823 new_backend_url_to_just_provide_most_recent_activity

### LINKED ISSUE #1823: New backend URL to just provide most recent activity
Our backend provides the `logs` end-point for a wargame:
image

It return _all_ the player activity for that wargame - which is used by scientists to analyse Serge usage.

This end-point is also used by the `PlayerLog` modal window, to show current/recent player activity.  To do this, it downloads all the logs, then processes them to find the most recent activity for each player.

As the player log grows, this will be a larger and larger data download.  So, IMHO we need a different end-point on the backend:
`/:wargame/:dbname/logs-latest`

This will process the backend data to only have the most recent activity for each user, and return that smaller dataset.

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🧰 Issue

Fixes #1823 

## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by IanMayo** [comment]
Thanks @lilitkarapetyan 

One change:
> It returns all the player activity for that wargame - which is used by scientists to analyse Serge usage.

Some scientists still need the `logs` API.  So, we need to ***add*** `logs-latest`, and not to replace it.

**Comment by lilitkarapetyan** [comment]
> 



> Thanks @lilitkarapetyan
> 
> One change:
> 
> > It returns all the player activity for that wargame - which is used by scientists to analyse Serge usage.
> 
> Some scientists still need the `logs` API. So, we need to _**add**_ `logs-latest`, and not to replace it.

Hi Ian,, I will work on it now

File: packages/server/providers/couchdb.js [hunk 1]
```diff
@@ -224,7 +224,7 @@ const couchDb = (app, io, pouchOptions) => {
       res.status(404).send({ msg: 'Wrong Player Name', data: null })
     }
 
-    const db = new CouchDB(couchDbURL(databaseName))
+    const db = new PouchDB(databaseName, pouchOptions)
```
Comment (IanMayo) [inline packages/server/providers/couchdb.js]: @lilitkarapetyan - shouldn't this use `CouchDB` ?

Comment (lilitkarapetyan) [reply packages/server/providers/couchdb.js]: Hello Ian, yes we should use CouchDB.

**Review by IanMayo — APPROVED** [review]
Working. Tested on CouchDb

**Pull request merged**